src/pages/[productSlug]/tutorials/[...tutorialSlug]/index.tsx

```diff
@@ -33,6 +34,11 @@ async function getStaticProps({
 	GetStaticPropsResult<TutorialPageProps>
 > {
 	const { productSlug, tutorialSlug } = params
+
+	if (!getIsBetaProduct(productSlug)) {
```

Comment:
I thought this would have been handled by the `getStaticPaths` setup 🤔

Reply:
Not necessarily, cause we have `fallback: 'blocking'` it'll try to run `getStaticProps` when someone requests a new path.

Reply:
ohhh